official/benchmark/logger.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Why have this here instead of in utils/logging? I'm not opposed, but we should have reasons, given the similarity of purpose.

Reply:
Also, nit: since we have `utils`, and will probably eventually have `models`, and will certainly have multiple benchmarks, how do you feel about naming this package `benchmarks` instead of the singular `benchmark`?

Reply:
My original intention is to put all benchmark related code into official/benchmark, but not the benchmark themselves. I will have other code like upload to bigstore and other libs in future change. I think they are more logically grouped together. 

I could move those code to officials/utils/benchmark if you prefer